CHANGELOG-0.5.md

```diff
@@ -1,3 +1,34 @@
+# Release notes for v0.5.1
+
+[Documentation](https://docs.k8s.io/docs/home)
+# Changelog since v0.5.0
+
+## Changes by Kind
+
+### Feature
+
+- Add `process_start_time_seconds` metric. ([#106](https://github.com/kubernetes-csi/external-resizer/pull/106), [@Jiawei0227](https://github.com/Jiawei0227))
+- Adding configurable QPS and Burst to k8s client and use a separate client for leader election. ([#91](https://github.com/kubernetes-csi/external-resizer/pull/91), [@RaunakShah](https://github.com/RaunakShah))
+- Pods watch is an expensive operation. Hence make the feature of handling volume in use errors optional, by registering pod informer based on a feature gate ([#89](https://github.com/kubernetes-csi/external-resizer/pull/89), [@saikat-royc](https://github.com/saikat-royc))
+
+### Uncategorized
+
+- Build with Go 1.15 ([#98](https://github.com/kubernetes-csi/external-resizer/pull/98), [@pohly](https://github.com/pohly))
+- Publishing of images on k8s.gcr.io ([#84](https://github.com/kubernetes-csi/external-resizer/pull/84), [@pohly](https://github.com/pohly))
```

Comment:
Oh hm we need to backport some more changes to actually make this work.

Reply:
I think it's going to be too much effort to backport the cloudbuild changes and fix the current multiarch build failure. I propose that we release these patches using the old travis method, and then copy the images over to gcr